apps/i18n/common/en_us.json

```diff
@@ -1488,6 +1488,10 @@
   "incubator_codebridge_python_desc": "Try out Code.org's first Python Code Editor.",
   "incubator_codebridge_python_project_button": "Create Your Own Project",
   "incubator_codebridge_python_title": "Codebridge: Python",
+  "incubator_genai_title": "Exploring Generative AI",
+  "incubator_genai_desc": "Try our new course that allows students to explore and create with generative AI, developing critical thinking and ethical awareness.",
+  "incubator_genai_course_button": "View Course",
+  "incubator_genai_marketing_button": "Learn More",
```

Comment:
This file seems to use an inconsistent mix of `snake_case`, `camelCase`, and `both_ofThemCombined` keys.  I have a slight preference for `camelCase` (which is also recommended for JSON keys [here](https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format)) but maybe it makes sense to stay consistent with other `incubator_` keys.

Reply:
Yeah, I decided to maintain consistency with the other keys within the incubator subpage but it be a point to bring up in our engineering weekly.